### PR TITLE
README.md was corrected.

### PR DESCRIPTION
**Pull request by saikro-student**
README file were corrected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1 @@
-# vita
```
Comment (saikro) [inline README.md:1]: It's good.

File: README.md [hunk 2]
```diff
@@ -1,2 +1 @@
...
-'vita' is an abbreviation for Vital Introduction In Algorithms.
+'vita' is an abbreviation for Vital Introduction To Algorithms.
```
Comment (saikro) [inline README.md]: It's good. Moreover, could you provide a wider description of the project?

**Comment by saikro** [comment]
It's good. Could you add a wider description of the project?

**Comment by saikro-student** [comment]
I'm going to modify #1 and introduce the changes asked by @saikro.

**Comment by saikro-student** [comment]
It was done.

**Pull request merged**